tests/foreman/ui_airgun/test_contentview.py

```diff
@@ -1102,6 +1389,82 @@ def test_positive_add_package_filter(session, module_org):
         assert expected_packages == actual_packages
 
 
+@tier2
+def test_positive_add_package_inclusion_filter_and_publish(session, module_org):
+    """Add package to inclusion content views filter, publish CV and verify
+    package was actually filtered
+
+    :id: 58c32cb5-1392-478e-807a-9c023d5ca0ea
+
+    :expectedresults: Package is included in content view version
+
+    :CaseLevel: Integration
+    """
+    filter_name = gen_string('alpha')
+    repo_name = gen_string('alpha')
+    package1_name = 'cow'
+    package2_name = 'bear'
+    create_sync_custom_repo(module_org.id, repo_name=repo_name)
+    repo = entities.Repository(name=repo_name).search(
+        query={'organization_id': module_org.id})[0]
+    cv = entities.ContentView(organization=module_org, repository=[repo]).create()
+    with session:
+        session.contentviewfilter.create(cv.name, {
+            'name': filter_name,
+            'content_type': FILTER_CONTENT_TYPE['package'],
+            'inclusion_type': FILTER_TYPE['include'],
+        })
+        session.contentviewfilter.add_package_rule(
+            cv.name, filter_name, package1_name, None, 'All Versions')
+        result = session.contentview.publish(cv.name)
+        assert result['Version'] == VERSION
+        packages = session.contentview.search_version_package(
+            cv.name, VERSION, 'name = "{}"'.format(package1_name))
+        assert len(packages) == 1
+        assert packages[0]['Name'] == package1_name
+        packages = session.contentview.search_version_package(
+            cv.name, VERSION, 'name = "{}"'.format(package2_name))
+        assert not packages
+
+
+@tier2
+def test_positive_add_package_exclusion_filter_and_publish(session, module_org):
```

Comment:
Isn't this also a candidate for removal? It seems like the scenario with exclude filter is also covered in test_positive_update_exclusive_filter_package_version

Reply:
I thought about that, but we have slightly different scenario. We are working with different packages and different rule conditions, so I decided to leave both test cases